app/assets/stylesheets/modules/sitter.scss

```diff
@@ -5,7 +5,8 @@
 .main-head {
   font-family: $dogstay-font-theme;
   font-size: 3em;
-  padding-bottom: 0.5em;
+  padding-top: 0.5em;
```

Comment:
Properties should be ordered font-family, font-size, padding-bottom, padding-top, text-align